quarkus-workshop-super-heroes/super-heroes/rest-heroes/pom.xml

```diff
@@ -25,14 +25,15 @@
 
     <properties>
         <compiler-plugin.version>3.8.1</compiler-plugin.version>
```

Comment:
Maybe update to `3.11.0`?

Reply:
I think I'd like to keep version changes in a separate changeset. I want to focus on introducing pact here, not on improving everything in the workshop. Otherwise the changesets just get big and harder to digest/review. I've made a note to do another PR with the compiler and surefire version updates.